src/model/page-config-resolver.js

```diff
@@ -283,13 +283,180 @@ class JsonLdParser {
   }
 }
 
+/**
+ * Class the describes the microdata found in the page
+ */
+class Microdata {
```

Comment:
Do we still really need this class? It looks like we'd operate totally great w/o it?